tests/lib/test_router.py

```diff
@@ -1,5 +1,6 @@
 import datetime
```

Comment:
Currently all HTTP method decorators are only tested in their sync variant and none are asserted in their async variant. This seems incomplete. Even though I do sometimes think the http method tests are quite a lot already but they do test 8 methods so I guess it makes sense.